src/ui/interaction/container/interactionScreen.tsx

```diff
@@ -55,7 +55,7 @@ interface Props
 const InteractionContainer = (props: Props) => (
   <React.Fragment>
     <StatusBar hidden />
-    <Container style={{ backgroundColor: Colors.greyDark }}>
+    <Container safeArea={false} style={{ backgroundColor: Colors.greyDark }}>
```

Comment:
If `safeArea` is optional, can't it be skipped here?

Reply:
`safeArea` is supposed to be true by default.

Reply:
Can be changed to `<Container withoutSafeArea>` if the suggestion in the comment above is implemented.